README.md

```diff
@@ -149,9 +149,9 @@ These methods are available on a Writer object:
   Connect to the nsqd specified.
 * `close()` <br/>
   Disconnect from nsqd.
-* `publish(topic, msgs, [callback])` <br/>
+* `publish(topic, msgs, [timeMs], [callback])` <br/>
   `topic` is a string. `msgs` is either a string, a `Buffer`, JSON serializable
-  object, a list of strings / `Buffers` / JSON serializable objects. `callback` takes a single `error` argument.
+  object, a list of strings / `Buffers` / JSON serializable objects. `timeMs` is the delay by which the message should be delivered. `callback` takes a single `error` argument.
```

Comment:
I believe there's an upper bound on how long the timeMs can be based on the server configuration. It would be a good idea to produce a warning or error if it's beyond the maximum configured time. This is based on memory so it needs to be confirmed.